Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStore.cs

```diff
@@ -1139,26 +1139,27 @@ public async Task<BoolResult> TrimBulkAsync(OperationContext context, IReadOnlyL
                         DateTime lastAccessTime = contentHash.LastAccessTime;
                         int replicaCount = 1;
                         DateTime? effectiveLastAccessTime = null;
+                        long size = 0;
+                        bool isImportantReplica = true;
+
+                        if (_localContentStore != null && _localContentStore.TryGetContentInfo(contentHash.Hash, out var contentInfo))

```

Comment:
In initialization compute buckets.
One option is to hard code them, like 30m, 2h, 4h, 12h, 24h, 48h, 3d, >3d

Reply:
Important should move the content to the next bucket (potentially configurable to move further).

Assuming we have logarithmic or manual buckets (that kind of logarithmic).

---
In reply to: [340377517](https://github.com/microsoft/BuildXL/pull/1122#discussion_r340377517) [](ancestors = 340377517)

Reply:
Cost does not affect buckets, only the position within one bucket.

---
In reply to: [340379919](https://github.com/microsoft/BuildXL/pull/1122#discussion_r340379919) [](ancestors = 340379919,340377517)